README.md

```diff
@@ -126,10 +126,12 @@ E.g. `foreman run rails server` or `foreman run rails console`.
     ``` user = User.last``` #or ```user = User.find_by(github_handle: "yourgithubhandle") ```
     ``` user.roles.create(name: "organizer") ```
     You can assign yourself other roles in the same way. If however you assign
-    yourself a student role AND another role, that may lead to unexpected behavior in the app. In that case, remove the student role.    
+    yourself a student role AND another role, that may lead to unexpected behavior in the app. In that case, remove the student role. 
```

Comment:
I think that instruction is probably still valid. I guess it depends on what you want to do, and there can probably be a role that is not covered by our initial fake data, so there would be no one to impersonate for that role.

We can definitely improve this part of the readme in a separate PR if necessary. For now, I'm merging this one 🎉